src/main/java/org/jabref/logic/formatter/bibtexfields/HtmlToLatexFormatter.java

```diff
@@ -35,29 +35,12 @@ public String format(String text) {
             return result;
         }
 
-        StringBuilder sb = new StringBuilder();
         // Deal with the form <sup>k</sup>and <sub>k</sub>
         result = result.replaceAll("<[ ]?sup>([^<]+)</sup>", "\\\\textsuperscript\\{$1\\}");
         result = result.replaceAll("<[ ]?sub>([^<]+)</sub>", "\\\\textsubscript\\{$1\\}");
-
-        // TODO: maybe rewrite this based on regular expressions instead
         // Note that (at least) the IEEE Xplore fetcher must be fixed as it relies on the current way to
         // remove tags for its image alt-tag to equation converter
-        for (int i = 0; i < result.length(); i++) {
-            int c = result.charAt(i);
-
-            if (c == '<') {
-                int oldI = i;
-                i = readTag(result, i);
-                if (oldI == i) {
-                    // just a single <, which needs to be kept
-                    sb.append('<');
-                }
-            } else {
-                sb.append((char) c);
-            }
-        }
-        result = sb.toString();
+        result = result.replaceAll("<[^>]{1,100}>", "");
```

Comment:
Why is it hardcoded till 100 characters?

Reply:
I think because MAX_TAG_LENGTH is set to 100.

Reply:
> Why is it hardcoded till 100 characters?

> I think because MAX_TAG_LENGTH is set to 100.

It is
```
private static final int MAX_TAG_LENGTH = 100;
```

Reply:
(On mobile, therefore quick idea)

Is this a class constant? Then integrate it into this string constant. 

Use pattern compile and introduce a class constant, please. Similar to other classes. 😅